### ISSUE TITLE
#error "Step/Dir UART not supported on Mechaduino yet"

### ISSUE THREAD

**Issue by kaloca** [Misfittech/nano_stepper#30]
The compilation is broken when using mechaduino hardware. How to not use Dir UART?

**Comment by coolio986** [comment]
@kaloca, This is a sanity check. You can change this line of code:

`#define USE_STEP_DIR_SERIAL`

to this:

`//#define USE_STEP_DIR_SERIAL`

**Comment by coolio986** [comment]
@trampas, can you add 

```
#ifndef MECHADUINO_HARDWARE
#define USE_STEP_DIR_SERIAL
#endif
```

**Comment by coolio986** [comment]
Fixed in b731396, closing
